pkg/bkop/backup.go

```diff
@@ -13,7 +13,7 @@ import (
 func (o operator) DumpFull(ctx context.Context, dir string) error {
 	args := []string{
 		fmt.Sprintf("mysql://%s@%s:%d", o.user, o.host, o.port),
-		"--passwords-from-stdin",
+		"-p" + o.password,
```

Comment:
In this case, I'd suggest embedding the password in the URI.
```go
		fmt.Sprintf("mysql://%s:%s@%s:%d", o.user, o.password, o.host, o.port),
```
https://dev.mysql.com/doc/refman/8.0/en/connecting-using-uri-or-key-value-pairs.html

Reply:
@ymmt2005 
I don't care either way but what's the pros of it?
cons: password should be escaped like:
```
	u := url.URL{
		Scheme: "mysql",
		Host:   fmt.Sprintf("%s:%d", o.host, o.port),
		User:   url.UserPassword(o.user, o.password),
	}
	args := []string{
		u.String(),
                ...
```

Reply:
@umezawatakeshi Fair enough.
Then, I'd withdraw my suggestion.

Reply:
I'm afraid we should've escaped the user name in the URI.

Reply:
@ymmt2005 
It seems that [constants](https://github.com/cybozu-go/moco/blob/v0.18.1/pkg/constants/users.go#L3-L13) are used as usernames in moco-backup container and they cannot be passed from outside. So, it's not necessary to escape username itself.